tests/js/testUtils.js

```diff
@@ -56,4 +56,12 @@ https://github.com/gpii/universal/LICENSE.txt
         });
     };
 
+    gpii.tests.utils.simulateKeyEvent = function (onElement, keyEvent, keyCode) {
```

Comment:
I think there are now two functions with the same name in this file. Maybe we should try to remove the other one and see if things still work.